app/components/govuk_component/notification_banner_component.rb

```diff
@@ -31,18 +31,19 @@ def render?
   end
 
   class Heading < GovukComponent::Base
-    attr_reader :text, :link_href, :link_text
+    attr_reader :text, :link_href, :link_text, :tag
 
-    def initialize(text: nil, link_text: nil, link_href: nil, classes: [], html_attributes: {})
+    def initialize(text: nil, link_text: nil, link_href: nil, tag: 'div', classes: [], html_attributes: {})
```

Comment:
Should we default to `p` as [used in the Nunjucks version](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/notification-banner/template.njk#L43-L46) if you just specify `text`? 🤔

Reply:
Yeah, that makes sense, will update!

Reply:
@frankieroberto done, thanks for the suggestion 😄